WinUIGallery/ControlPages/XamlCompInteropPage.xaml.cs

```diff
@@ -42,7 +43,9 @@ float GetDampingRatio()
         {
             if(DampingStackPanel.SelectedItem != null)
             {
-                return (float)Convert.ToDouble((DampingStackPanel.SelectedItem as RadioButton).Content);
+                // We need to specify the InvariantCulture since the decimal point depends on the
+                // system language and might parse "0.8" to 8 since the decimal point is a different character
+                return float.Parse((DampingStackPanel.SelectedItem as RadioButton).Content as string, CultureInfo.InvariantCulture);
```

Comment:
This should be: 
`return (float)Convert.ToDouble((DampingStackPanel.SelectedItem as RadioButton).Content, CultureInfo.InvariantCulture);`

Reply:
Good point, updated :)